lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -541,7 +541,19 @@ UsdMayaJobExportArgs::UsdMayaJobExportArgs(
           _Boolean(userArgs, UsdMayaJobExportArgsTokens->exportMaterialCollections))
     , exportMeshUVs(_Boolean(userArgs, UsdMayaJobExportArgsTokens->exportUVs))
     , exportNurbsExplicitUV(_Boolean(userArgs, UsdMayaJobExportArgsTokens->exportUVs))
-    , exportReferenceObjects(_Boolean(userArgs, UsdMayaJobExportArgsTokens->exportReferenceObjects))
+    , referenceObjectMode(_Token(
+          userArgs,
+          UsdMayaJobExportArgsTokens->referenceObjectMode,
+          UsdMayaJobExportArgsTokens->none,
+          { UsdMayaJobExportArgsTokens->attributeOnly, UsdMayaJobExportArgsTokens->defaultToMesh }))
+    , referenceObjectInterpolation(_Token(
+          userArgs,
+          UsdMayaJobExportArgsTokens->referenceObjectInterpolation,
+          UsdGeomTokens->varying,
+          { UsdGeomTokens->vertex,
+            UsdGeomTokens->constant,
+            UsdGeomTokens->faceVarying,
+            UsdGeomTokens->uniform }))
```

Comment:
Could you add this to the `std::ostream& operator<<(std::ostream& out, const UsdMayaJobExportArgs& exportArgs)` function as well?